packages/mobile/src/components/TokenBalance.test.tsx

```diff
@@ -0,0 +1,218 @@
+import { render } from '@testing-library/react-native'
+import * as React from 'react'
+import { Provider } from 'react-redux'
+import { HomeTokenBalance, FiatExchangeTokenBalance } from 'src/components/TokenBalance'
+import { LocalCurrencyCode } from 'src/localCurrency/consts'
+import { Currency } from 'src/utils/currencies'
+import { createMockStore, getElementText } from 'test/utils'
+import { mockTokenBalances } from 'test/values'
+
+const defaultStore = {
+  tokens: {
+    tokenBalances: mockTokenBalances,
+  },
+  localCurrency: {
+    preferredCurrencyCode: LocalCurrencyCode.USD,
+    fetchedCurrencyCode: LocalCurrencyCode.USD,
+    exchangeRates: {
+      [Currency.Dollar]: '1',
+      [Currency.Euro]: null,
+      [Currency.Celo]: null,
+    },
+  },
+}
+
+describe('FiatExchangeTokenBalance and HomeTokenBalance', () => {
+  it.each([HomeTokenBalance, FiatExchangeTokenBalance])(
```

Comment:
This looks like I added a lot of new tests, but the only thing I did was add the `it.each` to every test and remove the `expect(tree).toMatchSnapshot()` checks